src/sentry/api/endpoints/integrations/sentry_apps/internal_app_token/details.py

```diff
@@ -70,3 +71,27 @@ def delete(self, request: Request, sentry_app, api_token) -> Response:
         )
 
         return Response(status=204)
+
+
+# TODO: when the above endpoint is deprecated, simply move the convert method. This endpoint utilizes the id instead.
+@control_silo_endpoint
+class NewSentryInternalAppTokenDetailsEndpoint(SentryInternalAppTokenDetailsEndpoint):
+    owner = ApiOwner.INTEGRATIONS
+    publish_status = {
+        "DELETE": ApiPublishStatus.UNKNOWN,
```

Comment:
You can't add new endpoints with UNKNWON publish status. We locked it at some point to prevent more unknown getting added to the code base. Do you think this endpoint should be published at some point? Make it experiemental. Do you think it should stay private forever, make it private.

Reply:
Technically this isn't a new endpoint, it's still the "same" endpoint, it's just a different handler based on the regex. Realistically, once the FE moves off sending full token values, and ids instead, this "new endpoint" won't exist, as the "old endpoint" will just get the updated logic.

Reply:
I can mark it experimental for now I guess?

Reply:
I understand. But let's still have less unknown :P 

I actually moved a lot of token APIs to private thinking automating this kind of work probably doesn't make sense and we should leave it to frontend. Do you think we want to make this public at some point?

Reply:
Haha fair! I think it should be public, so let me try to tackle that in the next PRs after I migrate over the APIs first. This way people don't get confused on why one of them is public when the parent is unknown haha